### ISSUE TITLE
When copying and pasting in text with size large, the pasted text does not preserve size

### ISSUE THREAD

**Issue by sferoze** [quilljs/quill#1083]
If you copy a word that is size Large in Quill editor. And the paste the word on a new line. The pasted word does not preserve the size. It is pasted in normal size, when the format should be preserved. 

**Steps for Reproduction**
1. Visit quill example on homepage and use the 3rd example with the ability to change size in toolbar.
2. Make a word Large size
3. Copy that word to clipboard
4. Paste the word on a new line and it will not preserve the large formatting. 

**Expected behavior**:

Should preserve the format upon paste when copying text in quill editor

**Actual behavior**:

Does not preserve format, pasted text reverts to normal size

**Platforms**:

Chrome

**Version**:

Quill 1.1.3

**Comment by sferoze** [comment]
Its interesting. If you add a color to the large size text, and then copy and paste, then the size stays the same. Its only if its black text.

**Comment by bricksimpleseanmccaffery** [comment]
@jhchen Any update on this issue?

**Comment by kaptron** [comment]
By [adding a matcher](https://quilljs.com/docs/modules/clipboard/#addmatcher) I was able to get it to preserve the `ql-size-xx` format, like so:

```js
quill.clipboard.addMatcher('span, function(node, delta) {
  const match = node.className.match(/ql-size-(.*)/)
  if (match) {
    delta.map(op => {
      if (!op.attributes) op.attributes = {}
      // grab the size from `ql-size-{x}`
      op.attributes.size = match[1]
      return op
    })
  }
  return delta
});

```

**Comment by sebas0811buitrago** [comment]
I have a similar solution for react-quill 
```
const sizeMatcher = (node: HTMLElement, delta: Quill.Delta) => {
  const match = node.className.match(/ql-size-(.*)/);

  delta.map((operation) => {
    if (!operation.attributes) return {};
    if (!match) return operation;

    const size = match[1];
    operation.attributes.size = size;
    return operation;
  });

  return delta;
};
```
export const modules = {
  clipboard: {
    matchers: [['span', sizeMatcher]]
  }
};

**Comment by sakamotonekoo** [comment]
I tried to use ["*",sizeMatcher], it seems to be OK for many elements for now, not sure what would happen in the future

**Comment by benbro** [comment]
Fixed in [v2.0.0-rc.0](https://github.com/quilljs/quill/releases/tag/v2.0.0-rc.0)
https://v2.quilljs.com/
